infrastructure/src/main/java/org/corfudb/infrastructure/RemoteMonitoringService.java

```diff
@@ -7,11 +7,13 @@
 import lombok.Setter;
```

Comment:
I suggest improving the git-message. It's hard to understand what the patch is aiming to achieve with these messages

![image](https://user-images.githubusercontent.com/7818642/145912707-069bc426-10c9-436c-b280-9fba0f3d3174.png)

Reply:
I didn't know that anyone could see separate commits, so didn't pay attention to messages, since when merged everything will be squashed. All conversations happen in Github, would you like to use GitHub or you'd like to have nice history of commits in my branch?